tests/uitests/build.gradle.kts

```diff
@@ -42,3 +42,7 @@ dependencies {
     allLibraries()
     allFeatures()
 }
+
+tasks.withType<Test> {
+    enabled = project.hasProperty("runPaparazzi")
+}
```

Comment:
It seems that [the coverage is not computed](https://github.com/vector-im/element-x-android/actions/runs/4290584239/jobs/7474748317) if we exclude this module from the global `test` task. So I would remove this `runPaparazzi` trick.

Reply:
Then we'll have to run it twice to make it fail, or maybe use some other trick to detect a failure in verifyPaparazzi like verifying if the `failures` folder exists?

Reply:
Yes, it's a shame... let's run them twice for now...

Reply:
Paparazzi not verifying by default seems to be by design: https://github.com/cashapp/paparazzi/issues/701#issuecomment-1404295807 . The workaround to change this behavior could not work IIUC.